_maps/eosstation.json

```diff
@@ -0,0 +1,18 @@
+{
+	"version": 1,
+	"map_name": "EosStation",
+	"map_path": "map_files/EosStation",
+	"map_file": "Eosstation.dmm",
```

Comment:
```suggestion
	"map_path": "map_files/Eosstation",
	"map_file": "EosStation.dmm",
```

Reply:
changed

Reply:
The map_file still isn't changed
tbh should just rename the folder to EosStation for consistency